src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2066,7 +2114,7 @@ def create_functionapp_slot(cmd, resource_group_name, name, slot, configuration_
         update_slot_configuration_from_source(cmd, client, resource_group_name, name, slot, configuration_source,
                                               image, registry_password,
                                               registry_username,
-                                              docker_registry_server_url=docker_registry_server_url)
+                                              container_registry_url=docker_registry_server_url)
```

Comment:
also check if we can rename docker_registry_server_url to container_registry_server_url

Reply:
This docker_registry_server_url is inside functionapp, will have separate PR for them if renaming in functionapp is needed.